### PR TITLE
stav: Initial commit

### PR DESCRIPTION
**Pull request by bact**
I would like to request for a permanent ID "stav" on w3id.org.
Details are below. Thank you.

STAV: System Trustworthiness and Accountability Vocabulary
Accompanied repo at https://github.com/bact/stav

Contact:
Arthit Suriyawongkul
ADAPT Centre, Trinity College Dublin
arthit.suriyawongkul@adaptcentre.ie

### CODE REVIEW

File: stav/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+This [W3ID](https://w3id.org) provides a persistent URI namespace for System Trustworthiness and Accountability Vocabulary (STAV) resources.
+
+
+## About
+
+STAV is a vocabulary for system trustworthiness and accountability. Initial focus is on technical documentation and AI database registration as required in the drafts of EU AI Act.
+
+The canonical URL for STAV is https://w3id.org/stav.
+The namespace for STAV terms is https://w3id.org/stav#, the suggested prefix is stav.
+This document along with source and releases will be available at https://github.com/bact/stav/.
```
Comment (TallTed) [inline stav/README.md:11]:
```suggestion
* The canonical URL for STAV is https://w3id.org/stav.
* The namespace for STAV terms is https://w3id.org/stav#, the suggested prefix is stav.
* This document along with source and releases will be available at https://github.com/bact/stav/.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by bact** [comment]
Sorry, I think I should use the "Add suggestion to batch" instead, to make it one single commit.
Anyway, please kindly review. Thank you.

(if a squashed PR is more preferred, I can close this PR and start a new one as well. Thank you)

**Comment by davidlehn** [comment]
This PR only has a `README.md`.  Did you forget to add a `.htaccess` file to do redirections?

**Comment by bact** [comment]
> This PR only has a `README.md`. Did you forget to add a `.htaccess` file to do redirections?

Thank you. .htaccess redirection is now tested and added.

**Comment by bact** [comment]
Thank you. It works now.

**Pull request merged**